### ISSUE TITLE
[cli] Add app command

### ISSUE THREAD

**Issue by dramich** [rancher/rancher#13390]
Add commands for apps:

`ls`
`delete`
`install`
`rollback`
`upgrade`
`list-templates`
`show-template`
`show-app`

**Comment by deniseschannon** [comment]
Available with CLI v2.0.1-rc2

**Comment by adingilloRancher** [comment]
Tested all commands for apps 
```
ls
delete
install
rollback
upgrade
list-templates
show-template
show-app
```

Was able to install apps, delete, rollback, and upgrade apps.
